components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/src/main/java/org/wso2/micro/integrator/prometheus/publisher/util/MetricFormatter.java

```diff
@@ -0,0 +1,79 @@
+package org.wso2.micro.integrator.prometheus.publisher.util;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.Collector.MetricFamilySamples;
+import io.prometheus.client.Collector.Type;
+
+import java.util.Enumeration;
+
+/**
+ * Metric Formatter class to format Prometheus Metrics
+ */
+public class MetricFormatter {
+
+    /**
+     * Return the type of metric
+     *
+     * @param type Metric Prometheus type
+     */
+    private String typeString(Type type) {
+
+        switch (type) {
+            case GAUGE:
+                return "gauge";
+            case COUNTER:
+                return "counter";
+            case SUMMARY:
+                return "summary";
+            case HISTOGRAM:
+                return "histogram";
+            default:
+                return "untyped";
+        }
+    }
+
+    /**
+     * Return a formatted metric
+     *
+     * @param metricsList the Prometheus Metric from Prometheus Default Registry
+     */
+    public StringBuilder formatMetrics(Enumeration<MetricFamilySamples> metricsList) {
```

Comment:
Can't we use this as a static util method?

Reply:
Changed